sbkeys/generate-aws-sbkeys

```diff
@@ -212,10 +244,10 @@ gencert() {
   cert_arn="$(\
     aws acm-pca issue-certificate \
       --certificate-authority-arn "${ca_arn}" \
-      --template-arn arn:aws:acm-pca:::template/BlankEndEntityCertificate_APICSRPassthrough/V1 \
+      --template-arn arn:${aws_partition}:acm-pca:::template/BlankEndEntityCertificate_APICSRPassthrough/V1 \
       --csr "fileb://${key}.csr" \
       --api-passthrough "file://codesign.json" \
-      --signing-algorithm "SHA256WITHRSA" \
+      --signing-algorithm "${signing_algorithm}" \
```

Comment:
This is fine, but if we can pick a default that works everywhere, like `SHA512WITHRSA`, I'd be fine to just update the hard-coded value.

Reply:
Yes, I set the default to `SHA256WITHRSA` because that is what we used for our Gen-6.1A profile.